addons/mail/static/src/core/common/message_actions.js

```diff
@@ -97,6 +97,15 @@ messageActionsRegistry
         title: (component) => (component.state.showTranslation ? _t("Revert") : _t("Translate")),
         onClick: (component) => component.onClickToggleTranslation(),
         sequence: 100,
+    })
+    .add("toggle-attachment-selection", {
+        condition: (component) => component.message.attachments.length > 1,
+        icon: "fa-check-square-o",
+        title: _t("Toggle attachment selection"),
```

Comment:
I'd rename to just `Select attachments`. The "Cancel" part will be mostly covered by immediate button next to selectable attachment.

The message action item will barely being used so it can be kept as `Attachment selection`, or if we really feel having a dedicate UI for cancel then `Cancel selection` (it's pretty obvious this is attachment selection as the user choose to click on `Select attachments` in the 1st place)